.circleci/config.yml

```diff
@@ -20,22 +25,48 @@ jobs:
           command: npm run test:ci
       - run:
           name: Install codecov
-          command: npm install -g codecov
+          command: sudo npm install -g codecov
       - run:
           name: Upload Coverage Report to Codecov
           command: codecov
-      - run:
-          name: Run E2E tests on release ready code
-          command: |
-            if [ "${CIRCLE_BRANCH}" == "master" ]; then
-              npm run e2e
-            fi
+      - save_cache:
+          paths:
+            - ~/repo/node_modules
+          key: node-v1-{{ .Branch }}-{{ checksum "package-lock.json" }}
+      - persist_to_workspace:
+          root: .
+          paths: node_modules
+  e2e_test: cypress/run
+  deploy:
+    docker:
+      - image: mamolli/cypress_awscli
```

Comment:
hm how about:
orbs:
  aws-cli: circleci/aws-cli@0.1.13

question is do we need cypress + aws cli, or can we have seperate jobs for "just deployments" and "just cypress"

Reply:
Well, for sure we don't need cypress inside `mamolli/cypress_awscli`, but I don't know exactly how this deploy script [deploy.js](https://github.com/BuildIt-Poland/daily-routine/blob/d498bf3cf2a626d915d664ee6ce79f4fab5aca56/deploy.js) is tied to the docker image. I see it was configured by @mgolawski. But generally speaking I think there's no need to use `mamolli/cypress_awscli` and we can use smaller image... but such change should to be tested first.